README.md

```diff
@@ -20,5 +24,4 @@ The documentation also includes more complete examples on how to use this action
 - name: publish minepkg package
   uses: minepkg/action-publish@main
   with:
-    api-key: ${{secrets.MINEPKG_API_KEY}}
-```
+    api-key: ${{secrets.MINEPKG_API_KEY}}
```

Comment:
i think you messed up the readme here. the closing ticks are gone (```)

Reply:
fixed in last commit, I think